src/text-editor.coffee

```diff
@@ -1057,6 +1057,14 @@ class TextEditor extends Model
   deleteToBeginningOfWord: ->
     @mutateSelectedText (selection) -> selection.deleteToBeginningOfWord()
 
+  # Similar to above, but deletes back to the previous word boundary
```

Comment:
Since the methods on `Selection` are public, then I'd say mark these as `Extended` for consistency.